skill/locales/pt-BR.json

```
@@ -1,105 +1,124 @@
-      "samples": [
-        "{diga|fale} a lista",
-        "{liste|lista} os dispositivos",
-        "{diga|fale} a lista de dispositivos",
-        "listar dispositivos",
-        "dispositivos",
-        "diga os números dos dispositivos",
-	"quais dispositivos tenho"
-      ]
-    },
-    {
-      "name": "DevicePlayIntent",
-      "slots": {
-        "DEVICENUMBER": "AMAZON.NUMBER"
-      },
-      "samples": [
-        "{reproduza|toca|reproduzir|iniciar a reprodução de|escutar|quero escutar|toque} {música|spotify} no {dispositivo| número|dispositivo número|} {-|DEVICENUMBER}"
-      ]
-    },
-    {
-      "name": "DeviceTransferIntent",
-      "slots": {
-        "DEVICENUMBER": "AMAZON.NUMBER"
-      },
-      "samples": [
-        "{trocar|troque} o dispositivo  {dispositivo|número|dispositivo número|} {-|DEVICENUMBER}",
-        "{passar|transferir|transfere|transferencia da} {reprodução|música} do {dispositivo|número|dispositivo número|} {-|DEVICENUMBER}"
-      ]
-    },
-    {
-      "name": "GetTrackIntent",
-      "samples": [
-        "qual {canção|música|} {está tocando|cantando} {agora|}",
-        "que {canção|música} é {essa|}",
-	"qual é essa {canção|música}",
-	"qual é essa {canção|música|} que toca {agora|}"
-      ]
-    }
-  ]
+{
+	"intents":[
+            {
+                "name": "AMAZON.StopIntent",
+                "samples": []
+            },
+            {
+               "name":"PlayIntent",
+               "slots":[
+                  
+               ],
+               "samples":[
+                  "reproduzir",
+                  "tocar",
+                  "toque",
+                  "tocar música",
+                  "continuar",
+                  "continue"
+               ]
+            },
+            {
+               "name":"PauseIntent",
+               "slots":[
+                  
+               ],
+               "samples":[
+                  "pausa",
+                  "pause",
+                  "parar",
+                  "pare",
+                  "stop",
+                  "parar a música",
+                  "chega"
+               ]
+            },
+            {
+               "name":"SkipNextIntent",
+               "slots":[
+                  
+               ],
+               "samples":[
+                  "seguinte",
+                  "próxima",
+                  "pular"
+               ]
+            },
+            {
+               "name":"SkipPreviousIntent",
+               "slots":[
+                  
+               ],
+               "samples":[
+                  "anterior",
+                  "voltar"
+               ]
+            },
+            {
+               "name":"VolumeLevelIntent",
+               "slots":[
+                  {
+                     "name":"VOLUMELEVEL",
+                     "type":"AMAZON.NUMBER"
+                  }
+               ],
+               "samples":[
+                  "definir volume em {VOLUMELEVEL}",
```

```suggestion
                  "definir volume em {-|VOLUMELEVEL}",
```

A few places like this